### PR TITLE
Various fix: readme, makefile, pullapprove

### PR DESCRIPTION
**Pull request by runcom**

### CODE REVIEW

**Comment by runcom** [comment]
lgtm

File: Makefile [hunk 1]
```diff
@@ -19,3 +20,16 @@ lint:
...
 		exit 1; \
 	fi
+
+.PHONY: .gitvalidation
+
+EPOCH_TEST_COMMIT ?= e68e0e1110e64f906f9b482e548f17d73e02e6b1
+
+# When this is running in travis, it will only check the travis commit range
+.gitvalidation:
+	@which git-validation > /dev/null 2>/dev/null || (echo "ERROR: git-validation not found. Consider 'make install.tools' target" && false)
```
Comment (mtrmac) [inline Makefile]: `make install.tools` ?

Comment (runcom) [inline Makefile]: wrong, I meant make deps

File: .pullapprove.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+approve_by_comment: true
+approve_regex: '^(Approved|lgtm|LGTM|:shipit:|:star:|:\+1:|:ship:)'
+reject_regex: ^Rejected
+reset_on_push: false
+reviewers:
+  teams:
+  - image-maintainers
+  name: default
+  required: 2
```
Comment (mtrmac) [inline .pullapprove.yml]: 2?

**Comment by mtrmac** [comment]
How does this “fix” Travis? AFAICS this only adds extra requirements.

**Comment by runcom** [comment]
was just the name of the branch and github put that into the title, once merged it will disappear but ok...let's change it

File: .pullapprove.yml (hunk 2 above)
Comment (runcom) [inline .pullapprove.yml]: we are 3 maintainers right now, me you and brandon, 2 is the minimum consensus to make a PR ready to merge

**Pull request merged**